src/components/NavBar.vue

```diff
@@ -7,7 +7,7 @@
       <v-spacer></v-spacer>
       <v-btn outlined id="menu" v-if="!loggedIn" to="Login">Iniciar Sesión</v-btn>
```

Comment:
Deberías reemplazar `to="Login"` por `to="/login"`

Esto para que la url no se vea alterada.

Por ejemplo:
-Si utilizas `to="Login"` , la url es `http://localhost:8080/Login`
-Si utilizas `to="/login"` ,  la url es `http://localhost:8080/login` y concuerda con el formato de las demás url de la app.